src/internal/tester/suite.go

```diff
@@ -53,6 +53,7 @@ func NewUnitSuite(t *testing.T) *unitSuite {
 }
 
 type unitSuite struct {
+	//nolint:forbidigo
```

Comment:
these shouldn't be required as there's an exclude rule in .golangci.yaml file for it

Reply:
s'what I get for going by the ide and not the lint command.